react/src/components/Weather/CurrentWeather.js

```diff
@@ -5,28 +5,32 @@ import rain from '../../icons/desktop/weatherrain.png';
 import cloud from '../../icons/desktop/weathercloud.png';
 import { format } from 'date-fns';
 import { pl } from 'date-fns/locale'
-
+import { DisplayPressure,
+DisplayClouds,
+DisplayFeelTemp,
+DisplayWind,
+DisplayTemp,
+DisplaySmog, 
+DisplayHumidity,
+DisplaySunrise,
+DisplaySunset, } from './APIValidation'
+import { render } from 'react-dom';
```

Comment:
```
import { render } from 'react-dom';
import { format } from 'date-fns';
import { pl } from 'date-fns/locale'
import { DisplayPressure,
DisplayClouds,
DisplayFeelTemp,
DisplayWind,
DisplayTemp,
DisplaySmog, 
DisplayHumidity,
DisplaySunrise,
DisplaySunset, } from './APIValidation'
import cloud from '../../icons/desktop/weathercloud.png';
```

To tzw. kolejność importów. Najpierw react, później paczkowe rzeczy (w tym wypadku `date-fns`), później komponenty, później helpery (np. funkcje), później zdjęcia i style.